Source/Android/jni/MainAndroid.cpp

```diff
@@ -107,6 +107,12 @@ void Host_UpdateMainFrame()
 
 void Host_RequestRenderWindowSize(int width, int height)
 {
+  // Update touch pointer
+  JNIEnv* env;
+  IDCache::GetJavaVM()->AttachCurrentThread(&env, nullptr);
+  jmethodID callback =
+      env->GetStaticMethodID(IDCache::GetNativeLibraryClass(), "updateTouchPointer", "()V");
```

Comment:
This `jmethodID` should be cached in `IDCache`, because calling it repeatedly is slow.

Reply:
Fixed